changed the file name to .gitignore. In index.html added metategs, li…

Irina-pi opened the pull request:
…sts, tables, forms.

I probably got tangled in the branches. I made a mistake and tried to fix it. But this is not accurate.

Irina-pi committed:
changed the file name to .gitignore. In index.html added metategs, lists, tables, forms.

5tronciy commented on index.html:
```diff
@@ -2,13 +2,138 @@
       <a href="Hello/about.html">об авторе</a>
     </p>
+    <img src="./images/flower.jpg" alt="flowers">
+    <b>
```
`<ol>` is not phrasing content. Don't use `<b>` to wrap the blocks.
Read more [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/b)

Irina-pi replied:
I corrected.

5tronciy commented on index.html:
```diff
@@ -2,13 +2,138 @@
+    <form>
+      <fieldset>
+        <legend>My first form</legend>
+        <p>
```
The same as `<b>`. Use `<div>` to wrap the blocks.

Irina-pi replied:
Do I need to replace <p>?

Irina-pi replied:
I corrected.

5tronciy commented on index.html:
```diff
@@ -2,13 +2,138 @@
+          <label for="password">Password</label><br />
+          <input id="password" type="password" name="Password"/><br>
+          <label for="programminglevel">Your programming level</label><br />
+          <input id="programminglevel" type="range" name="programminglevel"/>
```
The default range doesn't match the task. Set it up yourself.

Irina-pi replied:
I corrected.

5tronciy commented on index.html:
```diff
@@ -2,13 +2,138 @@
+        <input id="answer-3" type="radio" name="question" value="3">
+        <label for="answer-3">Hyper Text Transfer Protocol</label><br />
+        <label for="comment">Comment</label><br>
+        <textarea id="comment" name="comment" placeholder="Comment"></textarea><br>
```
The length of text doesn't match the task.

Irina-pi replied:
I corrected.

Irina-pi replied:
I have added the cols="22" attribute to the textarea. did I do right?

5tronciy replied:
Hmm, sure it is not limit the textarea content to 500 characters.

5tronciy commented on index.html:
```diff
@@ -2,13 +2,138 @@
+        <label for="comment">Comment</label><br>
+        <textarea id="comment" name="comment" placeholder="Comment"></textarea><br>
+        <label for="select answer">What does HTML stand for?</label><br />
+        <select name="answer" id="select answer">
```
Notes about id:
The id name is case sensitive!
The id name must contain at least one character, cannot start with a number, and must not contain whitespaces (spaces, tabs, etc.).

Irina-pi replied:
I corrected.

5tronciy replied:
You didn't. Space is not allowed in id.

Irina-pi replied:
I understand! Thank you!

5tronciy reviewed (changes requested):
Looks good. But need to be worked at.

Irina-pi committed:
Replaced <P> tags with <div>.
Added min and max attributes to the <input type="range"> tag.
Fixed id name. shave added the cols="22" attribute to the textarea

5tronciy commented on index.html:
```diff
@@ -7,6 +7,7 @@
     <meta name="autor" content="Anton and Irina Bocharov">
     <meta name="vienport" content="width=device-width, initial-scale=1.0">
     <title>Моя первая страница</title>
+    <link rel="stylesheet" href="./styles/styles.css">
```
Create new branch for the next task pls.

Irina-pi replied:
ok
